.github/workflows/windows-acceptance-basic.yml

```diff
@@ -0,0 +1,415 @@
+# This workflow installs containerd and nerdctl from source and
+# simply runs through each of the nerdctl commands which should
+# support Windows to ensre they return a non-zero exit code.
+name: Windows Basic Acceptance
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  GO_VERSION: '1.19'
+  CONTAINERD_REF: 'main'
+  TEST_CONTAINER_IMAGE: 'k8s.gcr.io/e2e-test-images/busybox:1.29-2'
+
+jobs:
```

Comment:
Would it be better to have a test arg like docker.
```
go test -exec sudo -v ./cmd/nerdctl/... -args -test.target=docker
```

and if a test is windows compatible it will run automatically